python/pyspark/ml/clustering.py

```diff
@@ -1156,6 +1156,201 @@ def getKeepLastCheckpoint(self):
         return self.getOrDefault(self.keepLastCheckpoint)
 
 
+class _PowerIterationClusteringParams(JavaParams, HasMaxIter, HasPredictionCol):
```

Comment:
Why not directly add params into class `PowerIterationClustering`?

Reply:
@WeichenXu123 Thanks for your review. The params can be either inside class PowerIterationClustering or separate. I will move them back inside class PowerIterationClustering, to be consistent with the params in the other classes in clustering.